.env.example

```
@@ -5,9 +5,53 @@ OLLAMA_BASE_URL='http://localhost:11434'
+QDRANT_URL='http://localhost:6333'
+
+# Vector Database Cleanup Configuration
+VECTOR_DB_CLEANUP_INTERVAL_HOURS=24
+WEB_SEARCH_CACHE_MAX_AGE_DAYS=7
+WEB_SEARCH_CLEANUP_MAX_AGE_DAYS=30
+CHAT_FILE_CLEANUP_ENABLED=true
+COMPREHENSIVE_CLEANUP_ENABLED=true
+
+# For local dev
+ENABLE_SIGNUP=true
+CORS_ALLOW_ORIGIN=http://localhost:5173;http://localhost:8080
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)